src/activities/find-all-docmaps.test.ts

```diff
@@ -30,11 +30,11 @@ describe('parse-docmap-activity', () => {
     }));
 
     // Act
-    const result = await findAllDocmaps('http://somewhere.not.real/docmap/index');
+    const result = await findAllDocmaps([], 'http://somewhere.not.real/docmap/index');
 
     // Assert
     expect(result).toBeDefined();
-    expect(result?.length).toStrictEqual(1);
-    expect(result?.[0]).toMatchObject({ '@id': 'fake-docmap' });
+    expect(result?.docMaps.length).toStrictEqual(1);
+    expect(result?.docMaps?.[0]).toMatchObject({ '@id': 'fake-docmap' });
   });
 });
```

Comment:
Can we add a test for this new behaviour by passing in a hash of the fake docmap response?